Controller/AuthorizeController.php

```diff
@@ -242,9 +242,13 @@ protected function processSuccess(UserInterface $user, AuthorizeFormHandler $for
      * @param UserInterface $user
      *
      * @return string
+     *
+     * @deprecated since 1.5.2, to be removed in 2.0
      */
     protected function getRedirectionUrl(UserInterface $user)
     {
+        @trigger_error('deprecated', E_USER_DEPRECATED);
```

Comment:
`@trigger_error(sprintf('%s() is deprecated since version 1.6 and will be removed in 2.0', __METHOD__))`